self-core-impl/src/test/java/com/selfxdsd/core/GitlabRepoTestCase.java

```diff
@@ -65,4 +65,24 @@ public void canBeActivated() {
         Mockito.verify(project, Mockito.times(1)).resolve(Mockito.any());
     }
 
+    /**
+     * A GitlabRepo can return its webhooks.
+     */
+    @Test
+    public void returnsWebhooks() {
+        final Repo repo = new GitlabRepo(
+                Mockito.mock(JsonResources.class),
+                URI.create("http://localhost:8080/repos/mihai/test/"),
+                Mockito.mock(User.class),
+                Mockito.mock(Storage.class)
+        );
+        MatcherAssert.assertThat(
+                repo.webhooks(),
+                Matchers.allOf(
+                        Matchers.notNullValue(),
+                        Matchers.instanceOf(GitlabWebhooks.class)
+                )
+        );
```

Comment:
@criske Some indentation issues here, lines are indented by 8 spaces instead of 4, pls fix it :D